src/main/scala/org/bitcoins/core/protocol/Address.scala

```diff
@@ -96,12 +100,14 @@ object BitcoinAddress {
 object AssetAddress {
   def validate(assetAddress : String) : Boolean = {
     //asset addresses must have the one byte namespace equivalent to 19
-    //which ends up being 'a' in the ascii character set
-    val base58DecodeChecked : Try[Seq[Byte]] = Try(BitcoinSUtil.decodeBase58(assetAddress))
+    //which ends up being 'a' in the ascii character set.
+    //bytes size becomes 22
+    val base58DecodeChecked : Try[Seq[Byte]] = Try(Base58.decodeCheck(assetAddress))
     base58DecodeChecked match {
       case Success(bytes) =>
         if (bytes == null) false
```

Comment:
Why are we checking if something is null here? We should never have null values if we are using Scala libraries. We only need to check for null values if we are using java libraries